sidekiq/assets/configuration/spec.yaml

```diff
@@ -0,0 +1,10 @@
+name: Sidekiq
+files:
+- name: sidekiq.yaml
+  options:
+  - template: logs
```

Comment:
may need to resync config if you rebase because of this PR https://github.com/DataDog/integrations-core/pull/6121